README.md

```diff
@@ -76,6 +278,26 @@ make manifests
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+
+## Running the tests
+[Envtest](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest) is used to set up a similar environment for integration testing on the Reply URLs Operator by setting up and starting an instance of etcd and the Kubernetes API server, without kubelet, controller-manager or other components.
+
+All the automated tests can be ran by running the command below.
+```shell
+make test
+```
+
+## GitHub Workflows
+
+### Build and test workflow
+This workflow builds and tests the Operator code. When the tests are successful, the workflow builds a container image and pushes the image to a container registry.
+
+### Promote workflow
+Promotes the built image when the PR is closed and approved.
+
+### Tag code workflow
+Tags the code base when the config directory gets updated.
+
 ## License
```

Comment:
⛔ it's using the MIT license at the root which is our standard license why is this apache

also no need to duplicate license either just completely remove this or point to the license file